docs/application/dotnet/guides/device/attached-devices.md

```diff
@@ -215,18 +216,18 @@ To control LEDs on the device:
 
 To lock and unlock the CPU state:
 
--   Lock the power state with the `RequestCpuLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
+-   Lock the power state with the `RequestLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
 
-    The method locks the CPU for a specified time. After the given time (in milliseconds), the lock is unlocked. If the process is destroyed, every lock is removed.
+    The method locks the given PowerLock of the [Tizen.System.PowerLock](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.PowerLock.html) enumeration for a specified time. After the given time (in milliseconds), the lock is unlocked. If the process is destroyed, every lock is removed.
 
     ```
-    Power.RequestCpuLock(2000);
+    Power.RequestLock(PowerLock.Cpu, 2000);
     ```
 
--   Unlock the power state with the `ReleaseCpuLock()` method:
+-   Unlock the power state with the `ReleaseLock()` method:
```

Comment:
Question: Why is the method name changed to `ReleaseLock()`  method?

As seen in the https://github.com/Samsung/TizenFX/blob/master/src/Tizen.System/Device/Power.cs link `ReleaseCpuLock()` method is used.

Reply:
RequestCpuLock and ReleaseCpuLock is deprecated and I made RequestLock and ReleaseLock because it needs to provide lock and release lock for other lock types besides cpu